app/assets/src/components/ui/controls/checkbox.scss

```diff
@@ -1,37 +1,41 @@
-@import "../../themes/colors";
+@import "~styles/themes/colors";
 
-.idseq-ui.input.checkbox {
+.checkbox {
   display: flex;
   position: relative;
-  padding: 0px 0px 0px 30px;
+  padding: 0px 0px 0px 0px;
   cursor: pointer;
   -webkit-user-select: none;
   -moz-user-select: none;
   -ms-user-select: none;
   user-select: none;
   height: 28px;
   align-items: center;
+  justify-content: center;
 
   input[type="checkbox"] {
-    position: absolute;
     opacity: 0;
     cursor: pointer;
   }
 
   .checkmark {
-    position: absolute;
-    top: 6px;
-    left: 6px;
+    position: relative;
     height: 16px;
     width: 16px;
+    min-width: 16px;
```

Comment:
Why do we need `min-width` when we have `width`?

Reply:
Yeah, there checkbox would shrink if the window was small enough - but I decided to use `flex: none` instead.